pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -114,6 +114,11 @@ func (c *Cluster) Start() error {
 	daemonIDs := c.getDaemonIDs()
 	var deploymentsToWaitFor []*v1.Deployment
 
+	mgrsToSkipReconcile, err := mon.ToSkipReconcile(c.clusterInfo.Context, c.context, c.clusterInfo.Namespace, config.MgrType, AppName)
```

Comment:
SkipReconcile Function could be moved out of the `mon` package.  Suggesting to use someting like `controller.SkipReconcile(..).

Reply:
make sense, done